### ISSUE TITLE
tokens not minted after locking ETH->SOL

### ISSUE THREAD

**Issue by Gokko558** [solana-labs/oyster#544]
similar to:
[issue](https://github.com/solana-labs/oyster/issues/513)

Recently me and another individuals started depositing wrapped ust to wormhole v1. I used my own oyster instance.
Only a minority of the over 100 transactions were sent on SOL.
see https://etherscan.io/token/0xa47c8bf37f92abed4a126bda807a7b7498661acd?a=0xf92cd566ea4864356c5491c177a430c222d7e678

I looked at some of the SOL accounts submitting the VAA's and saw a lot of failed transactions.
eg [this tx](https://solscan.io/tx/32mE5psEbxCXa1nrhby1aE5iDCT5277in1pySbXLbAYVoQCsSBWuU6aMa5SAHTQDcGBXqWvuxCKWb1SkasfmkeMR)
Given the amount in limbo and the volatility of these assets (a failing algorithmic stablecoin), I'd like to sell them asap...

**Comment by Gokko558** [comment]
I don't want to be 'that guy' but given the latest luna proposal from Do Kwon , not having ust in my wallet on the time of a upcoming fork might leave me with 0.
see: https://agora.terra.money/t/terra-ecosystem-revival-plan/8701

**Comment by Gokko558** [comment]
[leoluk](https://github.com/leoluk) maybe you could help out?
I can pay for your time, as well as any other dev with history in this repo.
As per the similar issue I mentioned, the code for recovery already exists?

> Yes - the **funds are safe** and our team is working on completing the transactions. We definitely feel for you and it sucks to have funds in limbo! Safely replacing the funds would take longer than getting them unstuck. I'm just one engineer of many working on this project, but rest assured that everyone is working hard to clear out the backlog on v1 and v2.
> 
> The reason it's taking so long is we had to add new functionality to the core node software first. We can't just push a button, we first had to add the button. This takes time - and testing, review and then external audits, since it involves changes to a critical path in the code base. The audits were completed on Friday and we're now backporting it to v1 and rolling it out.(https://github.com/solana-labs/oyster/issues/513#issuecomment-1038974362)

**Comment by evan-gray** [comment]
Hi @Gokko558, I'm reaching out to some folks to see if they have any insight 🙂  hang in there. Out of curiosity, why did you decide to use the (deprecated) V1 instead of the V2 on [Portal](portalbridge.com) (or similar)?

**Comment by Gokko558** [comment]
Today I received 3 transactions, about 50 of the 500k ust, great!
When looking at some addresses that minted wust before, I can see a lot of PostVAA tx were made today, but most failed.

[solscan](https://solscan.io/tx/44AgtaoQEsRnFfm1k2mVZdSHnbG3NN6bYxGmbW2fjABqXLyqnH7MSnpedSHjpRUzzRNxKRDcThEb56PkM8XKWQoq)
>  Secp256k1 Program instruction
> > Program log: Instruction: VerifySignatures
> > Program log: Error: AlreadyExists
> > Program Wormhole  consumed 50270 of 200000 compute units
> > Program returned error: custom program error: 0xb

I don't know anything solana related but 'AlreadyExists' is a worrisome error... that sounds like it might require an oyster consensus breaking hardfork.

> Hi @Gokko558, I'm reaching out to some folks to see if they have any insight 🙂 hang in there. Out of curiosity, why did you decide to use the (deprecated) V1 instead of the V2 on [Portal](https://portalbridge.com) (or similar)?
Much appreciated. 

There was an arb opportunity for wust. As I sold most of my ust at an 800k USDT loss, I was eager to recoup any amount of that.

**Comment by ckeun** [comment]
hey @Gokko558, we are in the process of recovering these txns. i believe your largest transfer should be completed. 
https://explorer.solana.com/tx/MKkPcP6JNACbo4XJtVH3fivQfbiWWE8fouqngnST2eJC9z7vdte5C3Ssf96s3xKiugEW9pshBCaPdaU2TAYM8nd

just wanted to let you know that this is a **deprecated** bridge and it is not supported. so it's difficult to find the resources to fix issues. in the future, we suggest you use [Portal](https://portalbridge.com/) instead, which works better and has a lot more active support.

**Comment by Gokko558** [comment]
> hey @Gokko558, we are in the process of recovering these txns. i believe your largest transfer should be completed. https://explorer.solana.com/tx/MKkPcP6JNACbo4XJtVH3fivQfbiWWE8fouqngnST2eJC9z7vdte5C3Ssf96s3xKiugEW9pshBCaPdaU2TAYM8nd
> 
> just wanted to let you know that this is a **deprecated** bridge and it is not supported. so it's difficult to find the resources to fix issues. in the future, we suggest you use [Portal](https://portalbridge.com/) instead, which works better and has a lot more active support.

That's great news, peace has once again returned inside my head. I can pay for your time (or colleagues) if you want?
Following of my tx are still missing according to my bookkeeping
Txhash	Quantity
0x5fb7aa9f450b2a6e9f458a26c79b4d14754a872edf8b6343e09da00e0a956d74	10,002
0x9f15c9bd772a0b3a305a4c764ee9284a9103043cecfaae0a5f5421fb6345c78f	10,103
0xbf708c81bfeb08c9fd2fa4e2a9298df55ba9b477ee4131a3d4ece18be2aa6d66	9,683.06
0xdc133eab288f9e22b23d5a7234a6a56a5da2b648c79da56e57c779c4803062d9	7,800
0xe8374b3e172a4ead75ff94f5c0884166ae9da6ea1c945c211e09430e510cae4c	10,000.09
0x8bc48038e18c678a43ed374c2c3aca5363435a28495d0b19d1f895b00171a9ab	10,000.11
0x9b4a7dc1f79ab8162c37c6d2adfb524ef6b5209ecabda5c9db537693ee28a0b5	10,000.12
0x6dc145d5bfd010743d2332784cda764dd34e38a29f5f3ab875b182640308ec6e	10,000.13
0x1205e5bb352a821a5a2bd6c3ab91c614323dd5bc0e7849615e766a8546996112	11,752.14

**Comment by bagz11** [comment]
Gokko, thank you very much, I just now saw this topic! I'm sorry that I competed with you in this arb situation... I tried to run a Worm V1 copy locally for several days and it was at the same time with you that I succeeded, so matched. you are a REAL MAN that wrote not only your transactions, but even mine. Thank you so much and I shake your hand! I wish you find more arb situations like this :) Thanks to your topic, I received the lost wUST_V1 yesterday, I was very pleasantly surprised.

Also many thanks to the Developers "Oyster" for their support, and also would like to join the offer to pay for this restoration work!

**Comment by Gokko558** [comment]
> Gokko, thank you very much, I just now saw this topic! I'm sorry that I competed with you in this arb situation... I tried to run a Worm V1 copy locally for several days and it was at the same time with you that I succeeded, so matched. you are a REAL MAN that wrote not only your transactions, but even mine. Thank you so much and I shake your hand! I wish you find more arb situations like this :) Thanks to your topic, I received the lost wUST_V1 yesterday, I was very pleasantly surprised.
> 
> Also many thanks to the Developers "Oyster" for their support, and also would like to join the offer to pay for this restoration work!

nice catch! yeah making it run was not easy given the wrong instructions.
FYI I'm still missing some deposits so probably you are too.

**Comment by bagz11** [comment]
> nice catch! yeah making it run was not easy given the wrong instructions. FYI I'm still missing some deposits so probably you are too.

yes, absolutely right. Some of the translations are missing, but I hope the developers will help us restore everything.

**Comment by Gokko558** [comment]
Today all tokens deposits have been completed, great thanks to all contributors!
